core-services/release-controller/_releases/priv/release-ocp-4.14-ppc64le.json

```diff
@@ -8,13 +8,5 @@
     "pullSecretName": "source",
     "referenceMode": "source",
     "to": "release-ppc64le-priv",
-    "verify": {
```

Comment:
do we even need this file if we are removing the only test?

Reply:
No, we need the file because it dictates what is visible on the release controller.